backend/src/app.ts

```diff
@@ -57,6 +57,34 @@ app.post(
   }
 );
 
+/**
+ * User sign up/sign in
+ */
+app.post(
+  '/userLoginSignup',
+  [middleware.authenticateUser, middleware.getDB],
+  async (req: express.Request, res: express.Response) => {
+    // search for existing user
+    console.log(req.user);
+    let user = await getUser(req.db, req.user.oid);
+
+    // create if not found
+    if (user === undefined) {
+      const newUser: User = {
+        primaryKey: `USER#${req.user.oid}`,
+        sortKey: `METADATA#${req.user.oid}`,
+        ...req.body,
```

Comment:
Remember that we're only sending data through the bearer token, there is no req.body sent in the post request. So we can remove line 76